lib/appium.js

```diff
@@ -257,7 +257,11 @@ class AppiumDriver extends BaseDriver {
     if (isAppiumDriverCommand(cmd)) {
       return super.executeCommand(cmd, ...args);
     }
-
+    // getStatus command should not be put into queue. If we do it as part of executionCommand, it will be added to queue.
+    // There will be lot of status commands in queue during createSession command, as createSession can take up to or more than a minute.
+    if (cmd === 'getStatus') {
```

Comment:
is condition ever reached since isAppiumDriverCommand always returns _true_ for `cmd === 'getStatus'`?